docs/connector-development/config-based/tutorial/6-testing.md

```diff
@@ -0,0 +1,42 @@
+# Step 6: Testing
+
+We should make sure the connector respects the Airbyte specifications before we start using it in production.
+This can be done by executing the Source Acceptance Tests (SAT).
+
+These tests will assert the most basic functionalities work as expected and are configured in `acceptance-test-config.yml`.
+
+Before running the tests, we'll create an invalid config to make sure the `check` operation fails if the credentials are wrong, and an abnormal state to verify the connector's behavior when running with an abnormal state.
+
+Update `integration_tests/invalid_config.json` with this content
+
+```
+{"access_key": "<invalid_key>", "start_date": "2022-07-21", "base": "USD"}
+```
+
+and `integration_tests/abnormal_state.json` with
+
+```
+{
+  "rates": {
+    "date": "2999-12-31"
+  }
+}
+
+```
+
+You can run the acceptance tests with the following commands:
+
+```
+docker build . -t airbyte/source-exchange-rates-tutorial:dev
+python -m pytest integration_tests -p integration_tests.acceptance
```

Comment:
The following tests failed, even if I'm able to run incremental read + check successfully: 
```
       4 failed
         - airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py:301 TestConnection.test_check[inputs1]
         - airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_incremental.py:113 TestIncremental.test_two_sequential_reads[inputs0]
         - airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_incremental.py:145 TestIncremental.test_read_sequential_slices[inputs0]
         - airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_incremental.py:207 TestIncremental.test_state_with_abnormally_large_values[inputs0]
```

Reply:
I also have a lot of noise while running the tests:
<img width="1702" alt="Screen Shot 2022-08-09 at 17 13 51" src="https://user-images.githubusercontent.com/5551758/183688302-24adba27-aa89-44bc-92b8-2ce47524b309.png">

Reply:
I'm pretty sure the read and check errors are because of the API switch

I think there are 2 issues with the noisy logs:
1. The acceptance tests shouldn't be logging debug logs. @brianjlai do you know why we're always logging the declarative source to debug?
2. `DEBUG` is not a valid `AirbyteMessage` type. I think the the type [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/logger.py#L59) should be `LOG` @brianjlai does that sound right to you?